peekingduck/pipeline/nodes/model/yolact_edgev1/yolact_edge_files/detector.py

```diff
@@ -20,12 +20,11 @@
 from pathlib import Path
 from typing import Dict, List, Tuple
 
+import numpy as np
+import torch
 import torch.backends as cudnn
```

Comment:
This is outside of the purpose of this PR, but making a note here as it is important.

`torch.backends` comprises multiple sub-types, CUDNN being one of them.
c.f. https://pytorch.org/docs/stable/backends.html
This import should be `from torch.backends import cudnn` instead. Then lines 100-102 below would be correct.

Can consider whether to fix this here or in another PR.

Reply:
Will raise this as a separate issue